Assets/Scripts/WorldProperties.cs

```diff
@@ -62,15 +63,16 @@ public class WorldProperties : MonoBehaviour
         private const double EARTH_RADIUS = 6378137;
 
         //Unity to lat --> multiply by scale; lat to Unity --> divide by scale
-        public static float Unity_X_To_Lat_Scale = 10.0f;
-        public static float Unity_Y_To_Alt_Scale = 10.0f;
-        public static float Unity_Z_To_Long_Scale = 10.0f;
+        private static float Unity_X_To_Lat_Scale = 10.0f;
+        private static float Unity_Y_To_Alt_Scale = 10.0f;
+        private static float Unity_Z_To_Long_Scale = 10.0f;
 
         // Use this for initialization
         void Start()
         {
             selectedDrone = null;
             dronesDict = new Dictionary<char, Drone>(); // Collection of all the drone classObjects
+            dronesQueue = new Queue<Drone>();
```

Comment:
There was no apparent reason for the drone's dictionary. I didn't remove it in case someone else might need it in the future. I'll comment it out for now.

Reply:
Actually I'll just remove it

Reply:
actually, there's no need for any id's on the world prop or drones, I'll just assign them from ros manager